app/models/video.py

```diff
@@ -0,0 +1,26 @@
+from flask import current_app
+from app import db
+from sqlalchemy.orm import relationship
+
+
+class Video(db.Model):
+    video_id = db.Column(db.Integer, primary_key=True, autoincrement=True)
+    title = db.Column(db.String)
+    release_date = db.Column(db.String)
+    total_inventory = db.Column(db.Integer)
+    available_inventory = db.Column(db.Integer)
```

Comment:
Like with `Customer`, we can compute `available_inventory` with a relationship:
```rentals = db.relationship('Rental', back_populates='video', lazy=True)```

Then in `to_dict`,  the `available_inventory` is the total inventory minus length of the `rentals`.
```available_inventory": self.total_inventory - len(self.rentals)```